byteparse/jvm/src/test/scala/byteparse/LargeBmpIteratorTests.scala

```diff
@@ -0,0 +1,60 @@
+package byteparse
+
+import java.net.URL
+
+import fastparse.{IteratorParserInput, LoggedParsedInput}
+
+import utest._
+import fastparse.allByte._
+import BmpParser._
+
+object LargeBmpIteratorTests extends TestSuite {
+  val url = "https://raw.githubusercontent.com/lihaoyi/fastparse/master/byteparse/jvm/src/test/resources/lena.bmp"
```

Comment:
Could we read this off disk as an InputStream rather than reading off the web? Otherwise we'll end up in odd situations where the tests will pass on the PR but after you commit it the repo on github changes and the tests fail, or vice versa